contracts/messaging/gateway/FuelERC20Gateway.sol

```diff
@@ -36,6 +36,9 @@ contract FuelERC20Gateway is
     // Constants //
     ///////////////
 
+    /// @dev The admin related contract roles
+    bytes1 public constant DEPOSIT_TO_CONTRACT = bytes1(keccak256("DEPOSIT_TO_CONTRACT"));
```

Comment:
I still feel like a `bool` would suffice here, but I can live with this

Reply:
Honestly, this could be anything, even just a byte of 0. As long as it effects the length of the data sent, then the fuel side contract should be able to understand what's going on